### ISSUE TITLE
Testing instructions

### ISSUE THREAD

**Issue by liveFreeOrCode** [DataDog/dd-trace-py#1944]
I'm having trouble getting the testing to work. Are these instructions right by chance: https://github.com/DataDog/dd-trace-py#testing?

## Docker compose up

```
matt:~/dev/dd-trace-py$ docker-compose up -d
WARNING: The TMPDIR variable is not set. Defaulting to a blank string.
Creating network "dd-trace-py_default" with the default driver
Creating dd-trace-py_testrunner_1 ... 
Creating dd-trace-py_ddagent_1    ... 
Creating dd-trace-py_consul_1     ... 
Creating dd-trace-py_consul_1        ... done
Creating dd-trace-py_elasticsearch_1 ... done
Creating dd-trace-py_redis_1         ... done
Creating dd-trace-py_postgres_1      ... done
Creating dd-trace-py_moto_1          ... done
Creating dd-trace-py_ddagent_1       ... error
Creating dd-trace-py_rediscluster_1  ... 
Creating dd-trace-py_cassandra_1     ... done
Creating dd-trace-py_vertica_1       ... 
Creating dd-trace-py_rabbitmq_1      ... 
Creating dd-trace-py_rediscluster_1  ... done
Creating dd-trace-py_testagent_1     ... error
Creating dd-trace-py_ddagent5_1      ... 
Creating dd-trace-py_rabbitmq_1      ... done
WARNING: Host is already in use by another container

Creating dd-trace-py_vertica_1       ... done2663): Bind for 127.0.0.1:8126 failed: port is already allocated
WARNING: Host is already in use by another container

Creating dd-trace-py_ddagent5_1      ... done Bind for 127.0.0.1:8126 failed: port is already allocated

ERROR: for testagent  Cannot start service testagent: driver failed programming external connectivity on endpoint dd-trace-py_testagent_1 (a142baf0a5e647466bcbe427ab09f65d32c5569f115f60b3852718e99cee2663): Bind for 127.0.0.1:8126 failed: port is already allocated

ERROR: for ddagent  Cannot start service ddagent: driver failed programming external connectivity on endpoint dd-trace-py_ddagent_1 (905a2d727b24d3cceb72312558805a58d10ac99e5fb24193777f3c3507305002): Bind for 127.0.0.1:8126 failed: port is already allocated
ERROR: Encountered errors while bringing up the project.
matt:~/dev/dd-trace-py$ 
```

## Testrunner

If I try to run testrunner anyway:
```
matt:~/dev/dd-trace-py$ docker-compose run --rm testrunner
WARNING: The TMPDIR variable is not set. Defaulting to a blank string.
Creating dd-trace-py_testrunner_run ... done
root@docker-desktop:~/project# ls
CHANGELOG.md  LICENSE         LICENSE.BSD3  README.md    ddtrace             docs            releasenotes  scripts    setup.py  tox.ini
Dockerfile    LICENSE.Apache  NOTICE        conftest.py  docker-compose.yml  pyproject.toml  riotfile.py   setup.cfg  tests
root@docker-desktop:~/project# tox
bash: tox: command not found
root@docker-desktop:~/project# 
```

## Environment

WSL: Microsoft Windows [Version 10.0.19042.685]

**Comment by Kyle-Verhoog** [comment]
@liveFreeOrCode yeah the instructions are a little out of date unfortunately as we've been making some changes to our tests recently 😅. We typically don't `docker-compose up -d` all the containers (but this should be fixed in #1767 regardless).

As for using the testrunner, `tox`/`riot` are not installed in the base image. `scripts/ddtest` can be used to run a one-off command in the test container. Really, the easiest way to test without much hassle is through CI.

**Comment by liveFreeOrCode** [comment]
Excellent, okay I thought I was going nuts haha.  Appreciate the speedy response

**Comment by Kyle-Verhoog** [comment]
The docker-compose issue was resolved with #1767 and we have a card tracking the work to update our dev docs so I think it's safe to close this issue out 🙂
